tests/ui/test_ui.py

```diff
@@ -68,10 +69,17 @@ def test_left_column_view(self, view, mocker):
         pile = mocker.patch('zulipterminal.ui.urwid.Pile')
         return_value = view.left_column_view()
         menu_view.assert_called_once_with()
-        streams_view.assert_called_once_with()
+        expected_streams_view_calls = [
```

Comment:
wow

Reply:
Was this in reference to a particular line?

Reply:
line 72-78. This is my first time seeing this approach.